server/athenian/api/precompute/accounts.py

```diff
@@ -35,175 +35,191 @@
 
 async def main(context: PrecomputeContext, args: argparse.Namespace) -> None:
     """Precompute several accounts."""
-    log, sdb, mdb, pdb, rdb, cache, slack = (
-        context.log, context.sdb, context.mdb, context.pdb, context.rdb, context.cache,
-        context.slack,
+    time_to = datetime.combine(
+        date.today() + timedelta(days=1), datetime.min.time(), tzinfo=timezone.utc,
```

Comment:
Can you please return the original formatting here, every argument used to be on a separate line which was critical to quickly comprehend in this case.